sdk/identity/Azure.Identity/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## 1.4.0-beta.6 (Unreleased)
 
+### New Features
+
+- By default, the MSAL Public Client Client Capabilities are populated with "CP1" to enable support for [Continuous Access Evaluation (CAE)](https://docs.microsoft.com/azure/active-directory/develop/app-resilience-continuous-access-evaluation).
+This indicates to AAD that your application is CAE ready and can handle the CAE claim challenge. This capability can be disabled, if necessary, by either setting an `AppContext` switch named "Azure.Identity.DisableCP1" to `true` or by setting the environment variable;
+"AZURE_IDENTITY_DISABLE_CP1" to "true".
```

Comment:
I like to include the csproj snippet to enable the app switch so it's easier for customers to apply.

```xml
 <ItemGroup>
    <RuntimeHostConfigurationOption Include="Azure.Core.Pipeline.DisableHttpWebRequestTransport" Value="true" />
  </ItemGroup> 
```

Reply:
Thanks, I didn't know about this. Should this be a documented implementation guideline?

Reply:
I wouldn't mind it 👍